tst/testinstall/methwhy.tst

```diff
@@ -1,9 +1,11 @@
-gap> START_TEST("methwy.tst");
+gap> START_TEST("methwhy.tst");
 
 #
 gap> ApplicableMethod();
+#I  ApplicableMethod requires at least two arguments
```

Comment:
I really don't like this. No other function produces such "hint messages". In this case, I also don't see any value it adds over the usage error message below.